docs/en/latest/admin-api.md

```diff
@@ -51,16 +51,16 @@ For security reasons, please modify the default `admin_key`, and check the `allo
 
```

Comment:
why we need  modify this fine ?

Reply:
done! Already restored because of my IDE issue